collections/landsat-8-9-l2.yaml

```diff
@@ -1,20 +1,23 @@
-Name: Landsat - 8 L2
+Name: Landsat 8-9 L2
 Description: |
-  [Landsat -8 mission](https://www.usgs.gov/core-science-systems/nli/landsat/landsat-8?qt-science_support_page_related_con=0#qt-science_support_page_related_con) 
-  is the most recent satellite of the Landsat program and it carries OLI(Operational Land Imager) and TIRS(Thermal Infrared Sensor) instruments onboard. 
-  OLI acquires data in the visible, near and short wave infrared regions while TIRS acquires data in the thermal region to provide essential information
-  for monitoring our natural resources and environment.
-  [Landsat 8 Level 2 data ](https://www.usgs.gov/core-science-systems/nli/landsat/landsat-collection-2-level-2-science-products)
+  [Landsat 8-9 Level 2](https://www.usgs.gov/centers/eros/science/usgs-eros-archive-landsat-archives-landsat-8-9-operational-land-imager-and) collection includes both 
+  [Landsat 8](https://www.usgs.gov/landsat-missions/landsat-8?qt-science_support_page_related_con=0#qt-science_support_page_related_con) and the most recently launched 
+  [Landsat 9](https://www.usgs.gov/landsat-missions/landsat-9) satellites (provided by NASA/USGS), both carrying the Operational Land Imager (OLI) and 
+  the Thermal Infrared Sensor (TIRS) instruments, with 9 optical and 2 thermal bands. These two sensors provide seasonal coverage of the global landmass.
+
+  [Landsat 8-9 Level 2 data](https://www.usgs.gov/core-science-systems/nli/landsat/landsat-collection-2-level-2-science-products)
   from the most recently released [collection 2](https://www.usgs.gov/faqs/what-updates-are-being-made-landsat-collection-2?qt-news_science_products=0#qt-news_science_products), 
   provides atmospherically corrected Surface Reflectance and Surface Brightness Temperature products generated from Collection 2 Level-1 scenes that have been processed to Tier 1 or Tier 2.
-  Collection 2 level 2 data are available since February 2013 and new data are continously added usually within 24 hours after Level 1 scenes are available.
+  Collection 2 level 2 data are available since February 2013 for Landsat 8 and since January 2022 for Landsat 9 and new data are continously added usually within 24 hours after Level 1 scenes are available.
 Documentation: "[here](https://docs.sentinel-hub.com/api/latest/data/landsat-8-l2/)"
 Image: landsat-8-l2/landsat-8-l2.png 
 EOBrowser: https:sentinelshare.page.link/X3Hc
-Resolution: 30m
+Resolution: 30 m
```

Comment:
Not important, but other collections don't have space between number and unit

Reply:
Removed the space again for consistence but believe it's more correct to always use one there. Not super important though.